crates/primitives-traits/Cargo.toml

```diff
@@ -14,7 +14,7 @@ workspace = true
 [dependencies]
 # reth
 reth-codecs = { workspace = true, optional = true }
-
+reth-static-file-types.workspace = true
```

Comment:
we don't need these deps here and can remove this

this can be moved as dep to the reth-provider crate instead